src/main/webapp/search-script.js

```diff
@@ -107,7 +111,7 @@ async function verify() {
     const url = new URL("/login", window.location.origin);
     url.searchParams.set("token", token);
     const response = await fetch(url, {method:"POST"});
-    const userInfo = await response.json();
+    const userInfo = response.json();
```

Comment:
Why remove the await? Is response.json() actually not async here?

Reply:
I thought it wasn't, but I checked online and it definitely is haha, changed back

Reply:
Yeah it's a super unintuitive API ._.